pennylane/kernels/postprocessing.py

```diff
@@ -50,39 +51,137 @@ def displace_matrix(K):
     Returns:
         array[float]: Kernel matrix with negative eigenvalues offset by adding the identity.
     """
-    wmin = np.min(np.linalg.eigvals(K))
+    wmin = np.min(np.linalg.eigvalsh(K))
 
     if wmin < 0:
         return K - np.eye(K.shape[0]) * wmin
 
     return K
 
 
-# The below actually does the same as thresholding
-# import cvxpy as cp
+def flip_matrix(K):
```

Comment:
Why did you add that one?

Reply:
I think it makes sense to put it into the library though as the other paper used it